pkg/appgw/appgw_test.go

```diff
@@ -213,7 +213,7 @@ var _ = Describe("Tests `appgw.ConfigBuilder`", func() {
 			Name: &probeName,
 			ID:   to.StringPtr(appGwIdentifier.probeID(probeName)),
 			ApplicationGatewayProbePropertiesFormat: &n.ApplicationGatewayProbePropertiesFormat{
-				Protocol:                            n.HTTP,
+				Protocol:                            n.ApplicationGatewayProtocolHTTP,
```

Comment:
Is there anything else like "xxxxProtocolHTTP"? curious why it name the HTTP protocol per service/product.

Reply:
yes, quite a bunch
![image](https://user-images.githubusercontent.com/5294363/133310247-99d24ddf-3231-4bac-84a3-b6e4f73c3919.png)